tests/components/wallbox/test_switch.py

```diff
@@ -0,0 +1,153 @@
+"""Test Wallbox Lock component."""
+import json
+
+import pytest
+import requests_mock
+
+from homeassistant.components.switch import SERVICE_TURN_OFF, SERVICE_TURN_ON
+from homeassistant.components.wallbox import InvalidAuth
+from homeassistant.components.wallbox.const import CHARGER_STATUS_ID_KEY
+from homeassistant.const import ATTR_ENTITY_ID
+from homeassistant.core import HomeAssistant
+
+from tests.components.wallbox import entry, setup_integration
+from tests.components.wallbox.const import (
+    ERROR,
+    JWT,
+    MOCK_SWITCH_ENTITY_ID,
+    STATUS,
+    TTL,
+    USER_ID,
+)
+
+authorisation_response = json.loads(
+    json.dumps(
+        {
+            JWT: "fakekeyhere",
+            USER_ID: 12345,
+            TTL: 145656758,
+            ERROR: "false",
+            STATUS: 200,
+        }
+    )
+)
+
+
+async def test_wallbox_switch_class(hass: HomeAssistant) -> None:
+    """Test wallbox switch class."""
+
+    await setup_integration(hass)
+
+    state = hass.states.get(MOCK_SWITCH_ENTITY_ID)
+    assert state
+    assert state.state == "off"
```

Comment:
```suggestion
    assert state.state == "on"
```
Shouldn't this be on if the car is charging? Possibly the missing brackets for the `lower` call.

Reply:
Yes, it is fixed now, good catch